src/components/CartReducer.js

```diff
@@ -0,0 +1,35 @@
+import { useReducer } from "react";
```

Comment:
Minor thing, I would rename the file to `cartReducer.js`. When you start with an uppercase file name, you'd expect a React component.